tests/test_client.py

```diff
@@ -190,11 +190,12 @@ def test_auth_code_flow(self):
             self.assertLoosely(result, lambda: self.assertIn('access_token', result))
 
     def test_auth_code_flow_error_response(self):
-        with self.assertRaisesRegexp(ValueError, "state missing"):
+        assertRaisesRegex = getattr(self, "assertRaisesRegex", self.assertRaisesRegexp)
```

Comment:
Thanks for going the extra mile to remain backwards compatible with Python 2. However, this line would defeat the purpose in Python 3.11 because it would remove `assertRaisesRegexp`.

Reply:
Thanks for the catch. I actually copied it from other place in tests which has similar issue. It should be fixed now in both places.